dynawo/sources/Models/Modelica/Dynawo/Electrical/Injectors/InjectorIDQ_INIT.mo

```diff
@@ -0,0 +1,44 @@
+within Dynawo.Electrical.Injectors;
+
+/*
+* Copyright (c) 2015-2019, RTE (http://www.rte-france.com)
+* See AUTHORS.txt
+* All rights reserved.
+* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this
+* file, you can obtain one at http://mozilla.org/MPL/2.0/.
+* SPDX-License-Identifier: MPL-2.0
+*
+* This file is part of Dynawo, an hybrid C++/Modelica open source time domain simulation tool for power systems.
+*/
+
+model InjectorIDQ_INIT "Injector controlled by d and q current components idPu and iqPu"
```

Comment:
"Initialisation model for the injector controlled by d and q current components idPu and iqPu

Reply:
done